pyuploadcare/resources/file.py

```diff
@@ -179,9 +179,11 @@ def info(self):
             self.update_info()
         return self._info_cache
 
-    def update_info(self):
+    def update_info(self, with_appdata=False):
         """Updates and returns file information by requesting Uploadcare API."""
-        self._info_cache = self._client.files_api.retrieve(self.uuid).dict()
+        self._info_cache = self._client.files_api.retrieve(
+            self.uuid, include_appdata=with_appdata
```

Comment:
Different names?

Reply:
Do you suppose that is confusing naming? and straightforward passing the argument labeled `with_appdata` may be better?

Reply:
For me `include_appdata` is better name. Anyway, I just don't get why that names are different?

Reply:
I agree with you, `include_appdata` is more semantically cohesive!